wpilibj/src/main/java/edu/wpi/first/wpilibj/shuffleboard/Shuffleboard.java

```diff
@@ -0,0 +1,103 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2016-2018 FIRST. All Rights Reserved.                        */
```

Comment:
2018?

Reply:
That's what wpiformat generates

Reply:
Oops.  I wrote the wrong year.  Why 2016?

Reply:
Again, this is autogenerated by wpiformat. @calcmogul